### DISCUSSION TITLE
フォトライブラリの初期設定がわからないです。

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#139] [Question-original]
## (1) 実現したいこと
フォトライブラリの初期設定
## (2) 実現するために自分が試した内容とその結果
教科書を43ページまでやりました。Infoを開くところまでできています。
## (3) 調べた内容や資料
教科書：スタンプ（touchEvent）
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
教科書と表示が違うのでわからないです！
![image](https://user-images.githubusercontent.com/75009903/219290335-4b73a0aa-fcf2-4e9d-bbfa-0d72eccc0cdd.png)
![image](https://user-images.githubusercontent.com/75009903/219290513-969abfb7-1141-4250-8006-94aef70bcc9a.png)
## (5) GitHubのリンク
https://github.com/KOTARO1209/Stamp

**Comment by txxxxc** [comment]
以下の画像のような感じで一番上のプロパティをホバーしたら出てくるプラスボタンを押せば追加できるからやってみてもらっていい？
![image](https://user-images.githubusercontent.com/22112440/219292042-dbb00a06-dfbd-4219-b65b-1c284acb1daa.png)
